CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# dbt_hubspot_source v0.17.0
+
+## Breaking Change (`--full-refresh` required after upgrading)  
+- Introduced a new `category` column to the below models. This association field differentiates records by either HUBSPOT_DEFINED (default label) or USER_DEFINED (custom label) and was introduced to the connector in October 2024.
+  - `stg_hubspot__deal_company`
+  - `stg_hubspot__deal_contact`
+  - `stg_hubspot__merged_deal`
+  - `stg_hubspot__engagement_company`
+  - `stg_hubspot__engagement_contact`
+  - `stg_hubspot__engagement_deal`
+  - `stg_hubspot__ticket_company`
+  - `stg_hubspot__ticket_contact`
+  - `stg_hubspot__ticket_deal`
+  - `stg_hubspot__ticket_engagement`
+
+- This will be a breaking change and will require a `--full-refresh`.
```

Comment:
We can probably remove this since we already mention it in the title header.

Reply:
removed